src/modules/endpoint/sagas/loadCityContent.js

```diff
@@ -35,9 +35,13 @@ export default function * loadCityContent (
     call(loadLanguages, newCity, dataContainer, shouldUpdate)
   ])
 
+  // fetchResourceCache should always be called. Even if loadCategories, loadEvents, loadLanguages did not update the
+  // dataContainer this is needed. In case the previous call to fetchResourceCache failed to download some resources an
+  // other call could fix this and download missing files.
```

Comment:
As discussed in the comments of the issue, we would only want to fetchResourceCache if we update all other contents?! This makes updating content a lot simpler and it was the case before this task.

Reply:
Yes the issue description is invalid. I'll update it.
We have to call it each time you open a city because of the reasons I mention in the comment.

Reply:
As you did not comment on this one I take this as an: "OK"